editor/src/core/model/element-metadata-utils.ts

```diff
@@ -993,6 +997,19 @@ export const MetadataUtils = {
       maxSize: 1,
     },
   ),
+  isElementTypeHiddenInNavigator(path: ElementPath, metadata: ElementInstanceMetadataMap): boolean {
+    const VoidElementsToFilter = ['br', 'wbr']
```

Comment:
We should probably move this list to `dom-utils.ts`.

Reply:
fixed!